config/initializers/git_info.rb

```diff
@@ -0,0 +1,7 @@
+module Git
+    `which git`
+    if $?.success?
```

Comment:
I'd prefer that you required the `mkmf` library, and then used the `find_executable` method. It's cleaner.